lib/sms-sender-ids.json

```diff
@@ -0,0 +1,5 @@
+[
+  [ "CA", "16474909977" ],
```

Comment:
Virtual numbers that I'm using for testing.

As per [the documentation](https://docs.nexmo.com/messaging/sms-api/building-global-apps#senderID), legal values for the sender id depend on the region we're sending to. For the UK we can use an alphanumeric string up to 11 characters long, hence `Firefox`. For the USA it has to be a phone number, so I just plugged in the virtual phone number I'm using for USA testing, for now. For Canada, we should be able to use the short code that was already associated with out Nexmo account. However, in practice I found this got rejected by the network provider, so I've switched to the virtual number I'm using for testing there too.

For all of this, note the last item in the issue check-list: `Include the finalised sender ids`. It's one of the things we need to pin down before this can be merged.